version1/04 - Web services.md

```diff
@@ -468,6 +468,128 @@ Content-Type: */*
 Binary content
 ```
 
+# Full text web services
+
+Address:
+```
+https://{server}:{port}/rms/api/public/noark5/v1/full-text
+```
+
+Perform operations on index documents in a search index.
+- An **index document** is a set of fields and their corresponding values stored in a search index.
+- Each index document has a **doctype** which corresponds to a set of Noark 5 fields stored in the index document.
+- The supported doctypes and the object types (or fields) included in them are:
+
+| doctype            | Object types and fields                                                                                                                                                                                                                                                                                                |
+|:-------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
I'd suggest several improvements to the data in `Object types and fields`:
1. Rename the column to included Object types
2. There is no need to specify `all types` when we state `AbstraktRegistrering` and `AbstraktMappe`. This is explicitly mentioned in the `Object types.md`.
3. We must add `Dokumentversion` to the `Tekst` doctype
4. I would structure the object type list (or rather the comments in it) a little differently:
For `Tekst`: `Tekst (document text), Dokumentversjon (latest only), Dokument, AbstraktRegistrering (and its Noekkelord, Korrespondansepart, virksomhetsspesifikkeMetadata, secondary Klasse, and secondary Klassifikasjonssystem), AbstraktMappe (and its Noekkelord, virksomhetsspesifikkeMetadata, primary Klasse, secondary Klasse, secondary Klassifikasjonssystem, Sakspart, Arkivdel, and Arkiv)`
(and the same for the `Registrering` and `Mappe` doctypes.)

The primary reason I am suggesting 4. is because it is a little difficult for me to understand which satellite/secondary entities relate to which primary entities (Registrering, Mappe, etc.)

Reply:
Fixed.